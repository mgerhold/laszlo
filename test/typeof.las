println(typeof("Test"));
println(typeof(42));
println(typeof(true));
println(typeof(1..10));
println(typeof([1, 2, 3]));
println(typeof([]));
println(typeof(["Hello", "World"]));

for word in ["first", "second"] {
    println(typeof(word));
}

println(typeof([] + [1, 2, 3]));
println(typeof([] + []));
println(typeof([1, 2, 3] + []));
