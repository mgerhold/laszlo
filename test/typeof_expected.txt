String
I32
Bool
Range
Array[I32]
Array[?]
Array[String]
String
String
Array[I32]
Array[?]
Array[I32]
